### PR TITLE
Add /api/v2/import/token-info endpoint

### PR DESCRIPTION
**Pull request by nikitosing**
## Changelog
- add `icon_url` field to `tokens` table
- deprecate `ACCOUNT_AUTHENTICATE_ENDPOINT_API_KEY`
- add `SENSITIVE_ENDPOINTS_API_KEY`
https://github.com/blockscout/docs/pull/147
## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [x] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [ ] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: docker/Makefile [hunk 1]
```diff
@@ -687,6 +687,9 @@ endif
 ifdef EIP_1559_ELASTICITY_MULTIPLIER
 	BLOCKSCOUT_CONTAINER_PARAMS += -e 'EIP_1559_ELASTICITY_MULTIPLIER=$(EIP_1559_ELASTICITY_MULTIPLIER)'
 endif
+ifdef SENSITIVE_ENDPOINTS_API_KEY
```
Comment (vbaranov) [inline docker/Makefile:690]: I would suggest to move `_API` suffux to `API_` prefix => API_SENSITIVE_ENDPOINTS_KEY or API_SENSITIVE_ENDPOINTS_API_KEY, and suggest to add `API_` prefix to all Blockscout API related env variables, as we did for `API_RATE_LIMIT...`

Comment (nikitosing) [reply docker/Makefile]: Good suggestion. Done!

File: apps/block_scout_web/lib/block_scout_web/controllers/api/v2/import_controller.ex [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  action_fallback(BlockScoutWeb.API.V2.FallbackController)
+
+  def import_token_info(conn, %{"iconUrl" => icon_url, "tokenAddress" => token_address_hash_string} = params) do
+    with {:sensitive_endpoints_api_key, api_key} when not is_nil(api_key) <-
+           {:sensitive_endpoints_api_key, Application.get_env(:block_scout_web, :sensitive_endpoints_api_key)},
+         {:api_key, ^api_key} <- {:api_key, params["api_key"]},
+         {:format, {:ok, address_hash}} <- {:format, Chain.string_to_address_hash(token_address_hash_string)},
+         {:not_found, {:ok, token}} <- {:not_found, Chain.token_from_address_hash(address_hash, @api_true)} do
+      case token |> Token.changeset(%{icon_url: icon_url}) |> Repo.update() do
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/controllers/api/v2/import_controller.ex:19]: @nikitosing Please add validation, that this is valid URL before adding to changeset.

Comment (nikitosing) [reply apps/block_scout_web/lib/block_scout_web/controllers/api/v2/import_controller.ex:21]: Done

**Review by vbaranov — APPROVED** [review]

**Pull request merged**